linux_os/guide/system/network/network-nftables/set_nftables_loopback_traffic/ansible/shared.yml

```diff
@@ -0,0 +1,38 @@
+# platform = multi_platform_sle
+
+- name: Implement loopback rules
+  command: nft add rule inet filter input iif lo accept
+
+- name: Create rule to drop input IP address from loopback
+  command: nft insert rule inet filter input ip saddr 127.0.0.0/8 counter drop
+
+- name: Check if IPv6 is disabled
+  shell: |
```

Comment:
Maybe this `shell` task can be replaced by `lineinfile` and `assert`. Or `find`?